iquip/apps/builder.py

```diff
@@ -260,7 +264,7 @@ def run(self):
             response.raise_for_status()
             rid = response.json()
         except requests.exceptions.RequestException as err:
-            print(err)
+            logger.exception(err)
```

Comment:
`logger.exception("Failed to submit the experiment.")`

Reply:
You should print more detailed description in terms of what exception occurs. (Like Jiyong's suggestion!)

Previously, I just printed `err` because the logger was not implemented yet😅